selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -26,6 +26,83 @@
 #include "selfdrive/ui/qt/qt_window.h"
 #include "selfdrive/ui/qt/widgets/input.h"
 
+
+LongitudinalPersonality::LongitudinalPersonality() : AbstractControl("Driving Personality",
```

Comment:
can you add this as its own generic control type in selfdrive/ui/qt/widgets/controls.h?

Reply:
done